sources/packages/backend/apps/queue-consumers/src/processors/assessment/cancel-application-assessment.processor.ts

```diff
@@ -145,11 +146,19 @@ export class CancelApplicationAssessmentProcessor {
         transactionEntityManager,
       );
       await summary.info("Overawards rollback check executed.");
+      // Check if the assessment has no COE(s) which were declined.
+      const hasNoDeclinedCOE = assessment.disbursementSchedules.every(
```

Comment:
Completly minor for this scenario, but my first instinct would be to do the `some` as below. Is this way it would "stop" at the first currency instead of checking all. It is minor because it will be 1 or 2 items 😄 
```ts
assessment.disbursementSchedules.some((disbursement) => disbursement.coeStatus === COEStatus.declined)
```